src/game/Strategic/Strategic_Movement.cc

```diff
@@ -3109,7 +3109,7 @@ void AddFuelToVehicle(SOLDIERTYPE* pSoldier, SOLDIERTYPE* pVehicle)
 static void ReportVehicleOutOfGas(VEHICLETYPE const& v, const SGPSector& sMap)
 {
 	// Report that the vehicle that just arrived is out of gas
-	ST::string str = st_format_printf(gzLateLocalizedString[STR_LATE_05], pVehicleStrings[v.ubVehicleType], sMap.AsShortString());
+	ST::string str = st_format_printf(gzLateLocalizedString[STR_LATE_05], pVehicleStrings[v.ubVehicleType], sMap.y + 'A' - 1, sMap.x);
```

Comment:
Let's rather fix the format strings, so this can remain simple. Eg. in `src/game/Utils/_EnglishText.cc:3242` and the rest. If you want, you can convert it to ST syntax and then this line can also be changed to use `ST::format`, or just extract the c cstring from the short string to remain in the printf world.